pkg/builder/build.go

```diff
@@ -285,7 +286,15 @@ func (blder *Builder) doWebhook() error {
 		}
 	}
 
-	return err
+	err = conversion.CheckConvertibility(blder.mgr.GetScheme(), blder.apiType)
+	if err != nil {
+		log.Error(err, "conversion check failed", "GVK", gvk)
```

Comment:
i'm concerned that this error is going to be a bit spam-y for non-conversion cases (e.g. built-ins).  Maybe we should have an explicit opt-in on the builder?

Reply:
for posterity:

- error out if there's a partial implementation (hub w/o spokes, spokes w/o hub, partial set of spokes, multiple hubs, etc)
- log on success at `V(1).Info("conversioned enabled for kind", "kind", groupKind, "hub", hubVersion, "spokes", spokeVersions)` so that people can look for that line to debug too, or check for it's absence if they've failed to implement anything.

Reply:
> i'm concerned that this error is going to be a bit spam-y for non-conversion cases (e.g. built-ins).

If kube-apiserver sends us conversion requests for built-ins, it means there is a bug in kube-apiserver, right?